.github/workflows/release.yml

```
@@ -9,28 +9,23 @@ jobs:
     runs-on: windows-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: msys2/setup-msys2@v2
-    - name: Install dependencies
-      run: |
-         msys2 -c 'pacman -S --noconfirm libxslt base-devel'
     - name: Add msbuild to PATH
       uses: microsoft/setup-msbuild@v1.0.2
+    - name: Install msys2 dependencies
+      run: |
+        c:\msys64\msys2_shell.cmd -defterm -no-start -here -mingw64 -c 'pacman -S --noconfirm libxslt'
     - name: Build Generic64
-      shell: msys2 {0}
       run: |
-        CFLAGS="-O3 -g0 -march=ivybridge -mtune=native" ASMFLAGS="-march=ivybridge -mtune=native" make generic64/libXKCP.so
+        c:\msys64\msys2_shell.cmd -defterm -no-start -here -mingw64 -c "CFLAGS='-O3 -g0 -march=ivybridge -mtune=native' ASMFLAGS='-march=ivybridge -mtune=native' make generic64/libXKCP.so"
```

Does something like this not work?  I removed the shell.cmd and args which you might need but there are definitely some improvements in readability possible.

```suggestion
        make generic64/libXKCP.so
      env:
        CFLAGS: "-O3 -g0 -march=ivybridge -mtune=native"
        ASMFLAGS: "-march=ivybridge -mtune=native"
```

---

Note that you can get more readability by setting the environment variables above the run command.

---

It should work.

---

@ilkerhalil ?

---

@Rjvs It worked as Tyler suggested.